Modules/Office365DSC/DSCResources/MSFT_EXOSafeAttachmentPolicy/MSFT_EXOSafeAttachmentPolicy.psm1

```diff
@@ -0,0 +1,302 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Identity,
+
+        [Parameter()]
+        [ValidateSet('Block', 'Replace', 'Allow', 'DynamicDelivery')]
+        [System.String]
+        $Action = 'Block',
```

Comment:
SHould we default to Block or make this parameter mandatory?

Reply:
Same as above, I don't think it should be mandatory.  Block is the default value noted in the cmdlet docs.  Default values in DSC Resources are not present in $PSBoundParameters, so they don't really do anything unless the user specifies it in the configuration.  I only added them since they exist that way in the New/Set cmdlets.